Formula/bear.rb

```diff
@@ -19,21 +19,33 @@ class Bear < Formula
   depends_on "pkg-config" => :build
   depends_on "fmt"
   depends_on "grpc"
-  depends_on macos: :catalina
   depends_on "nlohmann-json"
   depends_on "python@3.9"
   depends_on "spdlog"
   depends_on "sqlite"
 
   uses_from_macos "llvm" => :test
 
+  on_macos do
+    depends_on "llvm" if MacOS.version <= :mojave
```

Comment:
Trying to make this a build-only dependency fails with
```
Undefined symbols for architecture x86_64:
  "std::__1::__fs::filesystem::__current_path(std::__1::error_code*)"
```
See https://github.com/Homebrew/homebrew-core/runs/2614926679#step:7:267 from https://github.com/Homebrew/homebrew-core/commit/027cafdd15a6571ee075544fc909495354c6ebc2.